README.md

```diff
@@ -40,6 +37,38 @@ To see all the grunt tasks that are available:
 grunt help
 ```
 
+### Using Angular-PatternFly In Your Application
+
+Note:
+
+1. Add Angular-PatternFly as a dependency for your project and you'll receive all the libraries you'll need:
+
+        $ bower install angular-patternfly --save
+
```

Comment:
I don't think this gives you everything since they still need to add in angular etc.